components/tikv_util/src/lib.rs

```diff
@@ -31,6 +31,7 @@ use std::time::Duration;
 use std::{env, thread, u64};
 
 use fs2::FileExt;
+pub use hex::{encode as hex_encode, encode_upper as hex_encode_upper};
```

Comment:
let's move these reexports to its own file, and let the script whitelist only that file.